app-modules/service-management/src/Providers/ServiceManagementServiceProvider.php

```diff
@@ -75,6 +77,13 @@ public function register(): void
     {
         Panel::configureUsing(fn (Panel $panel) => ($panel->getId() !== 'admin') || $panel->plugin(new ServiceManagementPlugin()));
 
+        TiptapEditor::configureUsing(function (TiptapEditor $component) {
+            $component
+                ->blocks([
+                    ServiceRequestTypeEmailTemplateButtonBlock::class,
+                ]);
+        });
```

Comment:
Instead of registering these blocks globally, they should probably be registered only on the TipTap instances that they should be used